Backends/Graphics3/OpenGL1/Sources/Kore/VertexBufferImpl.cpp

```diff
@@ -19,7 +19,7 @@ VertexBufferImpl::VertexBufferImpl(int count, int instanceDataStepRate) : myCoun
 #endif
 }
 
-Graphics3::VertexBuffer::VertexBuffer(int vertexCount, const Graphics4::VertexStructure& structure, int instanceDataStepRate)
+Graphics3::VertexBuffer::VertexBuffer(int vertexCount, const Graphics4::VertexStructure& structure, usage: Usage, int instanceDataStepRate)
```

Comment:
`usage: Usage` should probably be `Usage usage` as we're not in haxe land anymore?